### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by matty0ung** [Brewtarget/brewtarget#659] [General]
Lots of updates, bug-fixes, new features and improvements.

<hr /><em>This discussion was created from the release <a href='https://github.com/Brewtarget/brewtarget/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by b-a-p** [comment]
Hi @matty0ung,
your link directs to 404 page...

**Reply by matty0ung** [reply]
Oh yes, good point.  In my defence, that bit of text was auto-generated by GitHub!  Will fix.

**Comment by matty0ung** [comment]
Archiving this now since we're on the 4.0 release.
